tools/psa/tfm_ns_import.json

```diff
@@ -28,10 +28,6 @@
             "src_file": "install/export/tfm/src/tfm_ns_interface.c",
             "dst_file": "features/FEATURE_PSA/TARGET_TFM/TARGET_TFM_V8M/src/tfm_ns_interface.c"
         },
-        {
-            "src_file": "app/os_wrapper_cmsis_rtos_v2.c",
-            "dst_file": "features/FEATURE_PSA/TARGET_TFM/src/os_wrapper_cmsis_rtos_v2.c"
-        },
```

Comment:
"Remove TF-M import of os_wrapper_cmsis_rtos_v2.c" should be done before or as part of the commit "Change OS Wrapper layer". This is a better conceptual organization because we could run the importer at every commit and expect no bugs added from the import list and our own modifications being out of sync. Importing at commit "Change OS Wrapper layer" in current organization would break by overwriting our changes.

Reply:
Fixed.